app/controllers/v0/messages_controller.rb

```diff
@@ -70,11 +70,11 @@ def reply
       message_params[:id] = message_params.delete(:draft_id) if message_params[:draft_id].present?
       create_message_params = { message: message_params }.merge(upload_params)
 
-      if message.uploads.present?
-        client_response = client.post_create_message_reply_with_attachment(params[:id], create_message_params)
-      else
-        client_response = client.post_create_message_reply(params[:id], message_params)
-      end
+      client_response = if message.uploads.present?
+                          client.post_create_message_reply_with_attachment(params[:id], create_message_params)
+                        else
+                          client.post_create_message_reply(params[:id], message_params)
+                        end
```

Comment:
I'd argue the former is more readable, depends what you're used to 😄 I think it's more Ruby-ish (Lisp/Smalltalk influences) and avoids duplication.